src/state/pstake/utils.ts

```diff
@@ -109,7 +109,8 @@ export function getExtraTokensWeeklyRewardRate(
   token: Token,
   tokenMultiplier: JSBI | undefined,
 ) {
-  const TEN_EIGHTEEN = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(18));
+  const png = PNG[token.chainId];
+  const TEN_EIGHTEEN = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(png.decimals));
```

Comment:
I just wanted to know that, can you please tell me exact reason for this ?

Reply:
We were divided by 1e18, which is the png decimal in avalanche/flare/songbird but in hedera we can't divide by that value since the png decimal is 8, this makes us divide the value by a bigger number and make that 0 appears due to the truncation of the numbers